tools/db_bench_tool.cc

```diff
@@ -2214,7 +2214,7 @@ class Benchmark {
       CompressionOptions opts;
       CompressionContext context(FLAGS_compression_type_e);
       CompressionInfo info(opts, context, CompressionDict::GetEmptyDict(),
-                           FLAGS_compression_type_e);
+                           FLAGS_compression_type_e, 0 /* sample_for_compression */);
```

Comment:
Can you add a gflags option to specify the value of `sample_for_compression` with 0 as default value? It would be useful to tell us the overhead by running db_bench.

Reply:
Fixed.